### PR TITLE
Task/skin add accounts

### PR DESCRIPTION
**Pull request by einsteinx2**
**Is this pull request adding a feature or fixing a bug?**  
task

**Does this pull request close an issue? If so, which one?**
#334

**What does this pull request do? What does it change?**
Finish skinning add account view (needs some design tweaking, but otherwise done)

### CODE REVIEW

File: Balance/Shared/Data Model/Sqlite Models/Source.swift [hunk 1]
```diff
@@ -38,4 +38,15 @@ enum Source: Int, CustomStringConvertible {
...
     }
+    
+    var helpUrl: URL {
+        switch self {
+        case .coinbase:  return URL(string: "https://coinbase.com")!
+        case .poloniex:  return URL(string: "https://github.com/balancemymoney/balance-open/wiki/Poloniex-Guide")!
+        case .gdax:      return URL(string: "https://github.com/balancemymoney/balance-open/wiki/Gdax-Manual")!
+        case .bitfinex:  return URL(string: "https://github.com/balancemymoney/balance-open/wiki/Bitfinex-Guide")!
+        case .kraken:    return URL(string: "https://github.com/balancemymoney/balance-open/wiki/Kraken-Login-Manual")!
+        case .ethplorer: return URL(string: "https://etherscanio.freshdesk.com/support/solutions/articles/16000046111-what-is-an-ethereum-address-and-how-to-do-i-get-one-")!
```
Comment (wolffan) [inline Balance/Shared/Data Model/Sqlite Models/Source.swift]: 👍

File: Balance/macOS/Frameworks/YRKSpinningProgressIndicator/YRKSpinningProgressIndicator.h [hunk 2]
```diff
@@ -0,0 +1,26 @@
+//
+//  YRKSpinningProgressIndicator.h
+//
+//  Copyright 2009 Kelan Champagne. All rights reserved.
+//
+
+@import Cocoa;
+
+
+@interface YRKSpinningProgressIndicator : NSView
```
Comment (wolffan) [inline Balance/macOS/Frameworks/YRKSpinningProgressIndicator/YRKSpinningProgressIndicator.h]: I prefer external dependencies to have tests and be from carthage :(

File: Carthage/Build/Mac/BalanceVectorGraphics.framework/Versions/A/Resources/Info.plist [hunk 3]
```diff
@@ -27,17 +27,17 @@
 	<key>DTCompiler</key>
 	<string>com.apple.compilers.llvm.clang.1_0</string>
 	<key>DTPlatformBuild</key>
-	<string>9B55</string>
+	<string>9C40b</string>
```
Comment (wolffan) [inline Carthage/Build/Mac/BalanceVectorGraphics.framework/Versions/A/Resources/Info.plist]: warning why are those changed?

File: Balance/macOS/Frameworks/YRKSpinningProgressIndicator/YRKSpinningProgressIndicator.h (hunk 2 above)
Comment (einsteinx2) [reply Balance/macOS/Frameworks/YRKSpinningProgressIndicator/YRKSpinningProgressIndicator.h]: This thing is a tiny UI component that stopped being updated years ago (still in Obj-C). In general I agree on dependencies but we need this for 1.0 and it's really simple.

File: Carthage/Build/Mac/BalanceVectorGraphics.framework/Versions/A/Resources/Info.plist (hunk 3 above)
Comment (einsteinx2) [reply Carthage/Build/Mac/BalanceVectorGraphics.framework/Versions/A/Resources/Info.plist]: Lol who knows why Xcode does anything ¯\_(ツ)_/¯
